examples/gallery/src/line-layer.html

```diff
@@ -12,20 +11,32 @@
         height: 100vh;
         margin: 0;
       }
+      #tooltip:empty {
+        display: none;
+      }
+      #tooltip {
+        font-family: Helvetica, Arial, sans-serif;
+        font-size: 11px;
+        position: absolute;
+        padding: 4px;
+        margin: 8px;
+        background: rgba(0, 0, 0, 0.8);
+        color: #fff;
+        max-width: 300px;
+        font-size: 10px;
```

Comment:
duplicate

Reply:
I thought this is the standard tooltip we are using for the examples. Should I change the css? @xintongxia